tutorials/3d/introduction_to_3d.rst

```diff
@@ -190,22 +190,22 @@ light to your objects.
 Cameras
 -------
 
-No matter how many objects are placed in 3D space, nothing will be
+No matter how many objects are placed in the 3D space, nothing will be
 displayed unless a :ref:`Camera <class_Camera>` is
-also added to the scene. Cameras can either work in orthogonal or
+also added to the scene. Cameras can work in either orthogonal or
 perspective projections:
 
 .. image:: img/tuto_3d10.png
 
-Cameras are associated with and only display to a parent or grandparent
+Cameras are associated with -- and only display to -- a parent or grandparent
 viewport. Since the root of the scene tree is a viewport, cameras will
 display on it by default, but if sub-viewports (either as render target
 or picture-in-picture) are desired, they need their own children cameras
 to display.
 
 .. image:: img/tuto_3d11.png
 
-When dealing with multiple cameras, the following rules are followed for
+When dealing with multiple cameras, the rules about to be mentioned are followed for
```

Comment:
That's a bit too emphatic IMO. To avoid the repetition, I'd suggest using "the following rules are applied for each viewport" instead. (or "are used")

Reply:
@akien-mga : How about "[...] the following rules are enforced for each viewport: [...]"?

Reply:
Sounds good, yes.

Reply:
@akien-mga : Cool. So I'm done with this one.